packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1276,3 +1277,387 @@ export enum ProgressLocation {
      */
     Notification = 15
 }
+
+export class ProcessExecution {
+    private _process: string;
+    private _args: string[];
+    private _options: theia.ProcessExecutionOptions | undefined;
```

Comment:
FYI: using `_` as a prefix for the private properties is against the coding guidelines https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names
Please, check the properties in other classes.